README.es.md

```
@@ -24,22 +24,24 @@ Idioma: Español | [English](README.md)
 
 ---
 
-## Contenido {#contenido}
+<a id="readme-es-contenido"></a>
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)